tests/integration/targets/iso_customize/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: install pycdlib
+  pip:
+    name: pycdlib
+  register: install_pycdlib
+
+- debug: var=install_pycdlib
```

Comment:
No need to do that, if we run the tests with `-vvv` we can see enough debug information to know whether it has worked or not.

Reply:
Done. Thanks.